styles/_base.scss

```diff
@@ -121,3 +121,13 @@ form {
 .colEmphasis1 {
   background-color: $emphasis1;
 }
+
+#pageContainer {
```

Comment:
I think base is a good place for the main page container styles. If they get too big we can break them off into another file.